.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I don't think we should commit `.vscode` to this repository, but I do find this change to be useful. Maybe we should add this to `README.md` ?